Cyberduck/Support.md

```diff
@@ -99,14 +91,37 @@ To create a heap dump of the Cyberduck process on macOS (in case of excessive me
 
 Log output can be found in the `cyberduck.log` file in `%AppData%\cyberduck`.
 
+``````
+```````
+
+### Debug Logging
+
+````{tabs}
+```{group-tab} macOS
+
+To enable debug logging open a *Terminal.app* window and enter
+
+`defaults write ~/Library/Preferences/ch.sudo.cyberduck.plist logging debug`
+
+Reset the logging configuration with
+
+`defaults delete ~/Library/Preferences/ch.sudo.cyberduck.plist logging`
+
+Restart Cyberduck for any logging configuration change to take effect.
+
+```
+```{group-tab} Windows
 
-**Debug Log**<br/>
 Debug logging can be enabled with a [hidden setting](Preferences.md#hidden-configuration-options). As there is no user interface for this yet you need to add the setting manually. If not existing yet you have to create the file [`%AppData%\Cyberduck\default.properties`](FAQ.md#preferences-and-application-support-files-location) and then add the property as follows:
 
 `logging=debug`
 
-``````
-```````
+```
+````
+
+**Cyberduck 8.0 or later required**
+Alternatively, enable debug logging by ticking the corresponding checkmark in the Cyberduck Preferences Connection tab. You can reach the logging output by clicking on the Show button within the same Cyberduck Preferences section. The file is called cyberduck.log.
```

Comment:
Is the log reachability necessary at this point if it is stated in the section above?